### PR TITLE
Added GI, Profundity, Jabba. Split out fleet

### PR DESCRIPTION
**Pull request by dinkatoid**

### CODE REVIEW

File: src/data/events.ts [hunk 1]
```diff
@@ -79,28 +92,51 @@ export default new FarmCategory(
...
-                new FarmUnit('Slave I', 7),
-                new FarmUnit('IG-2000', 7),
-                new FarmUnit('Hound\'s Tooth', 7),
-                new FarmUnit('TIE Advanced x1', 7),
-                new FarmUnit('Imperial TIE Bomber', 7),
-                new FarmUnit('Imperial TIE Fighter', 7),
-            ],
-            [],
-        ),
+        // new Farm(
```
Comment (benbaxter) [inline src/data/events.ts:95]: nit: The Farm is still available in git's history and in src/data/fleets.ts. There is no need to keep around commented code.

Thanks for putting together this PR!

**Comment by dinkatoid** [comment]
I did this cause I had split out the fleet section into its own thing and I wasn't sure how you would feel about that. Is that OK, or do you want it mixed into events? I can remove the commented part and fix it up and send in an updated request.
________________________________

**Comment by benbaxter** [comment]
I do not have write permission to this repo, I was just leaving a
suggestion to clean up the code so the PR is in a final state to be
submitted.

(I was looking to add these events myself and saw you beat me to it! 😁)

**Review by Pelemane — DISMISSED** [review]
looks good

**Pull request closed without merging**